mlflow/langchain/databricks_dependencies.py

```diff
@@ -180,7 +251,10 @@ def _detect_databricks_dependencies(lc_model, log_errors_as_warnings=True) -> Li
     If a chat_model is found, it will be used to extract the databricks chat dependencies.
     """
     try:
-        dependency_list = list(_traverse_runnable(lc_model))
+        if isinstance(lc_model, AgentExecutor):
+            dependency_list = list(_extract_databricks_dependencies_from_agent(lc_model))
+        else:
```

Comment:
Also along with blocking the test, can we block this code as well so it doesn't run in version < 0.1.0 since the code will break and throw for the users.

Reply:
I will add it in, but theoretically the code wont break but it just wont work correctly. We have a try catch inside this to not parse UCFunctionToolkits. For Retriever tools, before 0.1.0 it was not using partial functions so its very hard to get the actual retriever.

But let me just add it in so that it will also just never run.